app/resources/admin.html.template

```diff
@@ -68,38 +68,99 @@
     //find all languages that the user specified with "Use?" checkboxes and
     //their associated titles.  Only allows titles from used languages.
     var used_languages = [];
-    var languages_to_titles = {}
+
     //there is a title row in the table, so the dropdowns are 1-indexed
-    for (var dropdown_index = 1; $('language_selection_' + dropdown_index) !=
+    for (var dropdown_index = 1; $('repo_titles_language_selection_' + dropdown_index) !=
          null; dropdown_index++) {
-      var language_code = $('language_selection_' + dropdown_index).value;
-      var language_title = $('language_title_' + dropdown_index).value;
+      var language_code = $('repo_titles_language_selection_' + 
+                             dropdown_index).value;
       if (language_code === 'unused') {
         continue;
       }
-      if (arr_contains(used_languages, language_code)) {
-        alert("You entered the same language (" + language_code + ") twice. " +
-              "That's not allowed.");
-        return false;
-      }
       used_languages.push(language_code);
-      languages_to_titles[language_code] = language_title;
+    }
+
+    var language_code_to_index_map = {};
+    for (var i = 0; i < used_languages.length; i++) {
```

Comment:
Can you combine this for loop with the one above? Please just ping me if what I'm saying is not clear.